### PR TITLE
Fix the real issues with consul/vault split

### PR DESCRIPTION
**Pull request by plukevdh**
Reverts articulate/docker-consul-template-bootstrap#28

- Adds `-e` so build fails
- Fixes bad archive path

### CODE REVIEW

File: install-centos.sh [hunk 1]
```diff
@@ -8,5 +8,5 @@ wget https://releases.hashicorp.com/consul-template/0.14.0/consul-template_0.14.
 unzip -d /usr/local/bin consul-template_0.14.0_linux_amd64.zip
```
Comment (tecnobrat) [inline install-centos.sh]: Shebang line needs to chnage in this file.

**Comment by fromonesrc** [comment]
@plukevdh I see at least one instance of consul template 0.14 in here (in centos install). We should probably get them all on the same version.

**Comment by plukevdh** [comment]
@fromonesrc What we should really do is combine the install scripts. I can work on that.

**Comment by plukevdh** [comment]
@fromonesrc this is now ported to combine the centos and debian distro installs. You can even test using the https://github.com/articulate/local-consul-template-test/ harness!

**Comment by fromonesrc** [comment]
Awesome @plukevdh, thanks for addressing this. Ship when you are ready.

**Comment by tecnobrat** [comment]
Needs a rebase now, but I love this approach.

**Comment by plukevdh** [comment]
https://www.youtube.com/watch?v=uvoUvDKh9SA

**Pull request merged**